### PR TITLE
Explainer for navigateEvent.sourceElement

### PR DESCRIPTION
**Pull request by natechapin**
Explainer for #225

### CODE REVIEW

**Review by domenic — APPROVED** [review]
This looks great, simple enough! I guess we could consider adding an example, but I don't think it's necessary.

Should we merge this now, or wait to try to finalize the name, or...?

File: README.md [hunk 1]
```diff
@@ -326,6 +326,8 @@ The event object has several useful properties:
 
 - `info`: any value passed by `navigation.navigate(url, { state, info })`, `navigation.back({ info })`, or similar, if the navigation was initiated by one of those methods and the `info` option was supplied. Otherwise, undefined. See [the example below](#example-using-info) for more.
 
+- `triggeringElement`: an `HTMLElement` or null, indicating what element (if any) initiated this navigation. If the navigation was triggered by a link click, the `triggeringElement` will be the `<a href="...">`. If the navigation was triggered by a form submission, the `triggeringElement` will be the [the element that sent the `submit` event to the form](https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent/submitter), or if that is null, the `<form>` being submitted.
```
Comment (domenic) [inline README.md:329]: Ah, we should mention the cases where it's censored.

**Review by domenic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (natechapin) [reply README.md]: Done!

**Comment by natechapin** [comment]
Updated the name and documented the cross-window case. I'm flexible as to when we declaring the naming discussion settled.

File: README.md [hunk 2]
```diff
@@ -326,6 +326,8 @@ The event object has several useful properties:
 
 - `info`: any value passed by `navigation.navigate(url, { state, info })`, `navigation.back({ info })`, or similar, if the navigation was initiated by one of those methods and the `info` option was supplied. Otherwise, undefined. See [the example below](#example-using-info) for more.
 
+- `sourceElement`: an `HTMLElement` or null, indicating what element (if any) initiated this navigation. If the navigation was triggered by a link click, the `sourceElement` will be the `<a href="...">`. If the navigation was triggered by a form submission, the `sourceElement` will be the [the element that sent the `submit` event to the form](https://developer.mozilla.org/en-US/docs/Web/API/SubmitEvent/submitter), or if that is null, the `<form>` being submitted. The `sourceElement` will also be null when a targeting a different `window` (e.g., `<a href="..." target="otherWindow">`).
```
Comment (domenic) [inline README.md:329]: Could it be a SVGElement, due to `<svg:a>`?? Seems like a good thing to add web platform tests for!!

Comment (natechapin) [reply README.md:329]: Yep, will add that to the CL

**Comment by natechapin** [comment]
I added `sourceElement` to the example of processing a form submission. Is that sufficient, or do you think it needs a standalone example?

**Comment by domenic** [comment]
That looks good! Just need to change `HTMLElement` to `Element` to deal with the SVG case.

**Comment by natechapin** [comment]
> That looks good! Just need to change `HTMLElement` to `Element` to deal with the SVG case.

Oops, done.

**Review by domenic — APPROVED** [review]

**Pull request merged**